src/Symfony/Component/Translation/Bridge/Crowdin/Provider/CrowdinProvider.php

```diff
@@ -0,0 +1,395 @@
+         * @see https://support.crowdin.com/enterprise/api/#operation/api.projects.translations.exports.post (Crowdin Enterprise API)
+         */
+        $response = $this->client->request('POST', sprintf('https://%s/projects/%d/translations/exports', $this->getEndpoint(), $this->projectId), [
+            'headers' => array_merge($this->getDefaultHeaders(), [
+                'Content-Type' => 'application/json',
+            ]),
+            'body' => json_encode([
+                'targetLanguageId' => $languageId,
+                'fileIds' => [$fileId],
+            ]),
+        ]);
+
+        if (Response::HTTP_NO_CONTENT === $response->getStatusCode()) {
+            throw new ProviderException(
+                sprintf('No content in exported file %d.', $fileId),
+                $response
+            );
+        }
+
+        if (Response::HTTP_OK !== $response->getStatusCode()) {
+            throw new ProviderException(
+                sprintf('Unable to export file %d translations for language %s.', $fileId, $languageId),
+                $response
+            );
+        }
+
+        $export = json_decode($response->getContent(), true);
+
+        return file_get_contents($export['data']['url']);
+    }
+
+    private function downloadSourceFile(int $fileId): string
+    {
+        /**
+         * @see https://support.crowdin.com/api/v2/#operation/api.projects.files.download.get (Crowdin API)
+         * @see https://support.crowdin.com/enterprise/api/#operation/api.projects.files.download.get (Crowdin Enterprise API)
+         */
+        $response = $this->client->request('GET', sprintf('https://%s/projects/%d/files/%d/download', $this->getEndpoint(), $this->projectId, $fileId), [
+            'headers' => array_merge($this->getDefaultHeaders(), [
+                'Content-Type' => 'application/json',
+            ])
+        ]);
+
+        if (Response::HTTP_OK !== $response->getStatusCode()) {
+            throw new ProviderException(
+                sprintf('Unable to download source file %d.', $fileId),
+                $response
+            );
+        }
+
+        $export = json_decode($response->getContent(), true);
+
+        return file_get_contents($export['data']['url']);
+    }
+
+    private function deleteFile(int $fileId): void
+    {
+        /**
+         * @see https://support.crowdin.com/api/v2/#operation/api.projects.files.delete (Crowdin API)
+         * @see https://support.crowdin.com/enterprise/api/#operation/api.projects.files.delete (Crowdin Enterprise API)
+         */
+        $response = $this->client->request('DELETE', sprintf('https://%s/projects/%d/files/%d', $this->getEndpoint(), $this->projectId, $fileId), [
+            'headers' => $this->getDefaultHeaders()
+        ]);
+
+        if (Response::HTTP_NOT_FOUND === $response->getStatusCode()) {
+            dump(sprintf('[Crowdin info] File ID "%d" Not Found for the project %d.', $fileId, $this->projectId));
```

Comment:
Please use an Exception here, and format the error text properly (no need of `[Crowdin info]`)